book-a-reading-room-visit.web/Views/Home/Availability.cshtml

```diff
@@ -89,7 +89,16 @@
                             <details open>
                                 <summary>Need a camera stand?</summary>
```

Comment:
@Brian-O-TNA , the headings in the existing boxes need changing too.
"Need a camera stand?" to "Camera stands"
"Are accessible seats available?" to "Accessible seating"